### PR TITLE
[Documentation][Entity Listeners] Mention lazy tag attribute

### PR DESCRIPTION
**Pull request by dmaicher**
The `lazy` tag attribute was not mentioned in the docs.

This might be superfluous in case https://github.com/doctrine/DoctrineBundle/issues/899 can/will be done

### CODE REVIEW

File: Resources/doc/entity-listeners.rst [hunk 1]
```diff
@@ -15,11 +15,7 @@ entity manager it should be registered with. Example:
                 class: \UserListener
                 tags:
                     - { name: doctrine.orm.entity_listener }
--
-                        name: doctrine.orm.entity_listener
-                        event: preUpdate
-                        entity: App\Entity\User
-                        entity_manager: custom
+
```
Comment (greg0ire) [inline Resources/doc/entity-listeners.rst]: This looks wrong, and seems related to #897

File: Resources/doc/entity-listeners.rst [hunk 2]
```diff
@@ -60,3 +57,34 @@ If you use a version of doctrine/orm < 2.5 you have to register the entity liste
 See also
 https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/events.html#entity-listeners
 for more info on entity listeners and the resolver required by Symfony.
+
+
+Lazy Entity Listeners
+---------------------
+
+You can use the `lazy` attribute on the tag to make sure the listener services are only instantiated when they are actually used.
```
Comment (greg0ire) [inline Resources/doc/entity-listeners.rst]: You need to double quote keywords, and I think linkes should be wrapped to 72 chars.

```suggestion
You can use the ``lazy`` attribute on the tag to make sure the listener
services are only instantiated when they are actually used.
```

File: Resources/doc/entity-listeners.rst (hunk 1 above)
Comment (dmaicher) [reply Resources/doc/entity-listeners.rst]: Indeed :+1:

**Comment by SenseException** [comment]
There's currently only a master branch documentation of DoctrineBundle on the Doctrine website, but should we put this addition to earlier supported versions of the docs?

**Comment by dmaicher** [comment]
Also related is this issue I created: https://github.com/doctrine/DoctrineBundle/issues/899

If we decide to implement that change then we might not even need to merge this PR.

File: Resources/doc/entity-listeners.rst (hunk 1 above)
Comment (alcaeus) [reply Resources/doc/entity-listeners.rst]: #897 was merged. Please rebase to remove the change here.

**Comment by alcaeus** [comment]
> This might be superfluous in case #899 can/will be done.

Since 1.10 will be around a bit longer, it would still make sense to merge this.

File: Resources/doc/entity-listeners.rst (hunk 1 above)
Comment (dmaicher) [reply Resources/doc/entity-listeners.rst]: Rebased. Keeping the new empty line is ok? Improves readability when editing it imho

**Comment by dmaicher** [comment]
This could be merged? :blush:

**Comment by alcaeus** [comment]
Ah, this should go into 1.10.x. Could you please change the base of the PR to 1.10.x and make sure no commits from master are in the PR? Thanks!

**Comment by greg0ire** [comment]
so `git rebase --onto 1.10.x master patch-1` (never tried it, but I saw that in the manual the other day)

**Comment by dmaicher** [comment]
Done :wink:

**Comment by alcaeus** [comment]
Okay. Build failure seems to be because travis-ci didn't choose the right branch when running the file. I've restarted the failing step to see if it will work this time.

**Comment by alcaeus** [comment]
Thanks @dmaicher!

**Pull request merged**